Domain/Do-Svyazi.User.Domain/Chats/Channel.cs

```diff
@@ -33,30 +34,59 @@ public class Channel : Chat
         CanDeleteChat = ActionOption.Disabled,
     };
 
-    public Channel(string name, string description)
+    public Channel(MessengerUser user, string name, string description)
         : base(name, description)
     {
+        var admin = new ChatUser
+        {
+            Role = _baseAdminRole,
+            ChatId = Id,
+            User = user,
+        };
+
+        Users.Add(admin);
         BaseAdminRole = _baseAdminRole;
         BaseUserRole = _baseUserRole;
     }
 
     public override void AddUser(ChatUser chatUser)
     {
-        throw new NotImplementedException();
+        if (chatUser is null)
+            throw new ArgumentNullException(nameof(chatUser), "User to set is null");
+
+        if (Users.Contains(chatUser))
+            throw new Do_Svyazi_User_InnerLogicException($"User {chatUser.User.Name} already exists in chat {Name}");
+
+        chatUser.Role = _baseUserRole;
+        Users.Add(chatUser);
```

Comment:
1. Сделай метод который из юзера делает ChatUser (можно в базовом классе)
2. Сообщения ошибок - я уже говорил об этом.

```suggestion
        if (user is null)
            throw new ArgumentNullException(nameof(user), "User {user.NickName} to add in chat {Name} is null");

        if (Users.Contains(chatUser))
            throw new Do_Svyazi_User_InnerLogicException($"User {user.NickName} already exists in chat {Name}");
        
        var chatUser = CreateChatUser(user);
        Users.Add(chatUser);
```

Аналогично в остальных методах